src/globalStyles.ts

```diff
@@ -0,0 +1,127 @@
+import styled from "@emotion/styled";
+import {
+  AppBar,
+  AppBarProps,
+  ButtonProps,
+  Card,
+  CardProps,
+  Container,
+  ContainerProps,
+  Tabs,
+  Typography,
+  TypographyProps,
+} from "@mui/material";
+import { Button } from "@mui/material";
+import { purple } from "@mui/material/colors";
+
+export const appStyle = {
+  a: {
+    textDecoration: "none",
+  },
+  body: {
+    margin: 0,
+    height: "100vh",
+    // background: "antiquewhite",
+  },
+};
+
+export const navbarStyle = { background: "#311e1e" };
+
+export const NavbarContainer = styled(AppBar)<AppBarProps>(() => ({
+  color: "black",
+  padding: "auto 30px",
+  borderBottom: "5px solid #CCCCCC",
+  boxShadow: "none",
+  background: "white",
+  ".MuiTypography-root": {
+    padding: "4px 10px",
+    color: "white!important",
+    borderRadius: "20px",
+    border: " 1px solid #6750A4",
+  },
+}));
+
+export const ButtonWrapper = styled(Button)<
+  ButtonProps & {
+    background?: string;
+    text_color?: string;
+    width?: string;
+    height?: string;
+  }
+>(({ theme, background, text_color, width, height }) => ({
+  border: "1px solid #6750A4",
+  padding: "auto 20px",
+  borderRadius: "40px",
+  width: width ? width : "inherit",
+  height: height ? height : "inherit",
+  color: text_color ? text_color : "",
+  backgroundColor: background,
+  "&:hover": {
+    backgroundColor: purple[700],
+    color: "white",
+  },
+}));
```

Comment:
You didn't need to create this. As pointed out in the instructions, you had to create a theme and specify the colors inside it. That would override styles in all MUI components including Button.
Only do it, if you have a very custom element to design.